x-pack/plugins/monitoring/server/alerts/base_alert.ts

```diff
@@ -377,4 +377,13 @@ export class BaseAlert {
   ) {
     throw new Error('Child classes must implement `executeActions`');
   }
+
+  protected createGlobalStateLink(link: string, clusterUuid: string, ccs?: string) {
+    const globalState = [`cluster_uuid:${clusterUuid}`];
+    if (ccs) {
+      globalState.push(`ccs:${ccs}`);
+    }
+    globalState.push('refreshInterval:(pause:!f,value:10000),time:(from:now-1h,to:now)');
```

Comment:
To keep it consistent with our defaults, otherwise it'll use timerange default of 15min, which will then persist to next state once they start navigating the app

Reply:
But our defaults are defined here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/plugin.ts#L134?

Reply:
Yeah, but it won't transfer to `url_state` unless we explicitly provide it (like I'm doing above). You can test this for yourself

Reply:
Sorry, but I'm still not following.

This is what I'm seeing which seems fine:
![Kapture 2020-10-02 at 14 31 51](https://user-images.githubusercontent.com/56682/94957816-70565180-04bc-11eb-9912-569c7fc07884.gif)

Reply:
Okay now try it with `?_g=(cluster_uuid:wuXG3QJKThmajyOWMx20hw)` (but your cluster id).`url_state` only adds a default state if one isn't present (which is your example above)

Reply:
When I add that, I don't see the extra filters in the global state, but the defaults are still in tact (`1h` time period, etc) so I don't really see the problem. The problem with the code, as is, is it creates two sources of truth for our defaults - if we wanted to change our default from `1h`, we'd have to know to come here too when I don't think that's really necessary.

Perhaps it's a bug that the global state doesn't update with the defaults when not all the keys are present (in the global state) which we could fix separately

Reply:
I think it falls back to whatever is set in your advance settings, or maybe in your case it detects what you were/are using in the current session (not sure, but I know they use `localStorage`). For me it defaults to 15min.

I don't think it's a bug in the state manager, since by not providing the time they expect a default behavior. Now whether the default scopes to our app or kibana global I think is a different discussion

In any case, I think we should keep this for uniformity. I'm not really worried about the defaults changing, since it hasn't for years (or since the beginning) and it probably won't anytime soon

Reply:
>  For me it defaults to 15min.

I can't seem to get this to happen locally. 

When I load the page without anything in the query string, it goes to `1h`. If I change it to `24h` and refresh, it stays `24h` (due to the [sync code](https://github.com/elastic/kibana/blob/master/src/plugins/data/public/query/state_sync/connect_to_query_state.ts#L176)). If I go click SM from the left navigation, it loads with the default `1h` again.

I think we need to figure out _why_ the default of `1h` isn't working for you, because it seems the code is designed to always use that default.

Reply:
There isn't much to figure out. Like I said it falls back to defaults, and for me it's:
![Screen Shot 2020-10-02 at 4 16 06 PM](https://user-images.githubusercontent.com/27822304/94966141-c16d4200-04ca-11eb-970c-cbc33a558a07.png)

Anyways, we're digressing here. These are all trivial things that are not really related to the scope of this PR and can be addressed/debated as a separate discussion/issue

Reply:
My defaults are the same:
<img width="911" alt="Screen Shot 2020-10-02 at 4 31 15 PM" src="https://user-images.githubusercontent.com/56682/94967262-b9ae9d00-04cc-11eb-92db-b7621eba5e85.png">

Reply:
It also relies on `localStorage`. Maybe try clearing it, or at least the following key:
`kibana.timepicker.timeHistory: "[{"from":"now-1h","to":"now"},{"from":"now-15m","to":"now"}]"`

Reply:
I agree it isn't related to this PR and we should defer the conversation, but we should have it at some point. I'm curious if there is an actual bug around this code that we need to fix.

Reply:
Hey catching up on this, I think introducing more than one place to define defaults is probably not a good idea and we should understand why it's needed before we introduce that kind of foot-gun. 

For now, if the goal is to get this alert in before FF, I recommend removing that line and then during the test plan phase try to understand why you both are seeing these different behaviors, and then document that back on this PR or in a new ticket. Hopefully it's not necessary at all, but if it is, we should export the defaults as a constant and then use it in both places at the very least.